storage/file.go

```diff
@@ -425,6 +425,10 @@ func (f FileServiceClient) GetFile(path string, fileRange *FileRange) (*FileStre
 	}
 
 	props, err := getFileProps(resp.headers)
+	if err != nil {
+		return nil, err
```

Comment:
Agreed the error would be vague.  It would probably help to have getFileProps() return a detailed message in its error object.  I don't think we need to fix it in this PR though.